src/ctim/generators/schemas/actor_generators.clj

```diff
@@ -11,28 +12,20 @@
 
 (def gen-actor
   (gen/fmap
-   (fn [id]
-     (complete
-      StoredActor
-      {:id id}))
-   (gen-id/gen-short-id-of-type :actor)))
+   (fn [[s id]]
+     (into s {:id id}))
```

Comment:
When keys are known `assoc` is faster than `into`. This remark in not a blocker of course, I generally forgot this optimisation myself most of the time.

~~~clojure
user=> (time (doseq [v (range 10000)] (assoc {:a "a"} :id v)))
"Elapsed time: 1.261498 msecs"
nil
user=> (time (doseq [v (range 10000)] (into {:a "a"} {:id v})))
"Elapsed time: 6.176702 msecs"
~~~